classes/system/heka/metric_collector/single.yml

```diff
@@ -14,3 +17,4 @@ parameters:
           database: ${_param:influxdb_database}
           username: ${_param:influxdb_user}
           password: ${_param:influxdb_password}
+          time_precision: ${_param:influxdb_time_precision}
```

Comment:
that's unfortunate because the 2 parameters must never be different and the deployer shouldn't deal with those details. But lets address it in another change since the depending PR is merged.